docs/product-getting-started-guide/src/main/asciidoc/decision-tables-proc.adoc

```diff
@@ -6,6 +6,9 @@ Decision tables are a way to generate rules driven from the data entered into a
 .Prerequisites
 The business rules have been defined. For more information, see <<_defining_business_rules>>.
 
+.Related information
+For more information on decision tables, see {URL_GUIDED_DECISION_TABLES}[_{GUIDED_DECISION_TABLES}_].
```

Comment:
Do we use "information on" or "information about"? I remember being corrected from "on" to "about" back in my IBM days. But the promised copy of the IBM Style Guide never reached me.

Reply:
(same applies to other instances of "information on")

Reply:
Great catch. Just checked the Style guide and it is "Information about". Never "information on". Fixing throughout.